docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -16,31 +17,42 @@ The Voyager Migration Assessment feature is specifically designed to optimize th
+- **Performance metrics**. Voyager analyzes performance metrics to understand workload characteristics and provide recommendations for optimization in YugabyteDB.
 
-- **Migration time estimation**: Offers a calculated estimate of the time needed to import data into YugabyteDB after export from the source database, helping with better migration planning. These estimates are calculated based on various experiments during data import to YugabyteDB.
+- **Migration time estimate**. An estimate of the time needed to import data into YugabyteDB after export from the source database. These estimates are calculated based on various experiments during data import to YugabyteDB.
 
-{{< warning title="Caveats" >}}
-Recommendations are based on experiments done on [RF3](../../../architecture/docdb-replication/replication/#replication-factor) setup on instance types with 4GiB memory/core against YugabyteDB v2024.1.
+{{< warning title="Caveat" >}}
+The recommendations are based on testing using a [RF3](../../../architecture/docdb-replication/replication/#replication-factor) YugabyteDB cluster on instance types with 4GiB memory per core and running v2024.1.
 {{< /warning >}}
 
-Note that for cases where it is not possible to provide database access to the client machine running voyager, you can gather the metadata from the source database using plain bash/psql scripts by voyager, and then use voyager to analyze the metadata directly.
+Note that for cases where it is not possible to provide database access to the client machine running Voyager, you can gather the metadata from the source database using plain bash/psql scripts provided with Voyager, and then use Voyager to analyze the metadata directly.
 
-### Sample Migration Assessment Report
+The following table describes the type of data that is collected during a migration assessment.
 
-A sample Migration Assessment Report is as follows:
+| Data | Collected | Details |
+| :--- | :-------- | :------ |
+| Application or user data  | No | No application or user data is collected. |
+| Passwords | No | The assessment does not store any passwords. |
+| Database metadata<br>schema,&nbsp;object,&nbsp;object&nbsp;names | Yes | Voyager collects the schema metadata including table IOPS, table size, and so on, and the actual schema. |
+| Database name | Yes | Voyager collects database and schema names to be used in the generated report. |
+| Performance metrics | Optional | Voyager captures performance metrics from the database (IOPS) for rightsizing the target environment. |
+| Server or database credentials | No | No server or database credentials are collected. |
+
+### Sample Migration Assessment report
+
+A sample Migration Assessment report is as follows:
 
 ![Migration report](/images/migrate/assess-migration.jpg)
 
-## Generate a Migration Assessment Report
+## Generate a Migration Assessment report
```

Comment:
Why doesn't this section match the layout of the other migration steps. Installation in section "Before proceeding..." for example. And the Prepare the source database section should use the Postgresql include file rather than repeating the same info.